opencl/CMakeLists.txt

```diff
@@ -21,7 +21,7 @@ set(OCL_LOADER_REPO
 # Repo tags/hashes
 
 set(OCL_HEADERS_TAG dcd5bede6859d26833cd85f0d6bbcee7382dc9b3)
-set(OCL_LOADER_TAG 792682ad3d877ab38573b997808bab3b43902b70)
+set(OCL_LOADER_TAG 9a3e962f16f5097d2054233ad8b6dad51b6f41b7)
```

Comment:
Could you please clarify why uplift of ocl loader version is required?

Reply:
The reason is this upstream commit: https://github.com/KhronosGroup/OpenCL-ICD-Loader/commit/4aa683775c4ee55594196c79def7a75103d75e4c

It refactors 'apppackage' driver loader from WinRT API to raw WinAPI. The problem was that this specific WinRT API was not present in MinGW, so it would need extra work to make it compile. Upstream update just lets avoiding this extra work.

I took the tip of the master for an update, but I believe any commit starting with https://github.com/KhronosGroup/OpenCL-ICD-Loader/commit/4aa683775c4ee55594196c79def7a75103d75e4c should be enough to fix the issue.

Reply:
I have made a separate PR: https://github.com/intel/llvm/pull/7864